### PR TITLE
✨UI | Social media username example

### LINKED ISSUE #402: Social media usernames: Show 'eg' in dialog
cc: @adamcogan @william-liebenberg 

When the dialog pops up to enter a social username, show an example under the text box

> e.g. 'matt-goldman'

image

### PR DESCRIPTION
**Pull request by AntPolkanov**
Fixes #402 

Screenshot_20230713-092138

### CODE REVIEW

File: src/SSW.Rewards/PopupPages/LinkSocial.xaml [hunk 1]
```diff
@@ -36,7 +36,7 @@
             <Grid Grid.Row="1"
                   VerticalOptions="Center"
                   HorizontalOptions="Center"
-                  HeightRequest="45"
+                  RowDefinitions="45,Auto"
```
Comment (matt-goldman) [inline src/SSW.Rewards/PopupPages/LinkSocial.xaml:39]: I'm not a huge fan of auto for height or width, especially when we know what the content is going to be. Can you make this a specified height?
Auto has performance impact.

**Review by matt-goldman — COMMENTED** [review]
Looks good, one small change I'd like to see.

File: src/SSW.Rewards/PopupPages/LinkSocial.xaml (hunk 1 above)
Comment (AntPolkanov) [reply src/SSW.Rewards/PopupPages/LinkSocial.xaml:39]: I agree that `Auto` is less performant than fixed sizes or `*`, especially if it's used in large lists.
The thing is that we don't know the font size on user's phones.

Comment (matt-goldman) [reply src/SSW.Rewards/PopupPages/LinkSocial.xaml:39]: As per our conversation, I agree we shouldn't worry about premature optimisation. Let's merge this as is.

**Review by matt-goldman — APPROVED** [review]
LGTM 🚀

**Pull request merged**